bundle/jsky.app.ot/src/main/java/jsky/app/ot/tpe/GemsGuideStarWorker.java

```diff
@@ -116,24 +120,35 @@ public void setInterrupted() {
     }
 
     /**
+     * AGS failed to run, so clear the AGS results.
+     */
+    public void clearResults() {
+        apply(tpe.getContext(), 0.0, GuideGroup.AutomaticInitial());
```

Comment:
Hmmm... this is a good point all around, and one that I had failed to consider.
I think it may apply in certain cases, such as when a `NoStarsException` is thrown by `GemsGuideStarWorker.findGuideStars`, which indicates that the search was not canceled and no stars were found. (Not sure if this happens, though, if there is no connectivity, but I am pretty sure that a `CatalogException` occurs in that case instead.)

So for now, I am going to propose that this method only be called when a `NoStarsException` is issued as the old results will no longer be valid, and in the other cases (`CatalogException`, `NoStarsException`, and general catch-all `Exception`), we maintain the current results instead of wiping them out.

This should be temporary, as when we switch to `BagsManager`, I do not believe that we will be using any auto search in `GemsGuideStarWorker` and will instead be going through the `AgsStrategy` instead.